pkg/codegen/codegen.go

```diff
@@ -281,47 +322,17 @@ func (b *Builder) makeTemplateParameters(ctx context.Context) (map[string]interf
 
 		"Box": boxConf,
 
-		"Manifest":  b.Manifest,
-		"Arguments": b.Manifest.Arguments,
+		"Manifest":  b.manifest,
+		"Arguments": b.manifest.Arguments,
 	}, nil
 }
 
 // FetchTemplate fetches a template from a git repository
 func (b *Builder) FetchTemplate(ctx context.Context, filePath string) (io.Reader, error) {
-	f, err := b.GitRepoFs.Open(filePath)
+	f, err := b.templateFS.Open(filePath)
 	return f, errors.Wrap(err, filePath)
 }
 
-// HasDeviations looks for deviation blocks in a file, returning true if they exist
-func (b *Builder) HasDeviations(_ context.Context, filePath string) bool {
```

Comment:
Deviations support is a big ol copy of the command parser and, honestly, I'd like to see us not support it in stencil proper. Maybe I'm idealistic here. If there's push back I'm happy to add it back (I'll cleanup the command parser so it's not duplicated at least)

Reply:
this is optimistic

Reply:
Will address when we get to it and redesign to be more visible